src/test/java/com/uwetrottmann/tmdb2/TmdbTestSuite.java

```diff
@@ -76,9 +87,55 @@ public static void initializeEnvironment() throws IOException, ParseException {
         unauthenticatedInstance = new BaseTestCase.TestTmdb(API_KEY);
         authenticatedInstance = new BaseTestCase.TestTmdb(API_KEY);
 
+        initializeData();
+
+        initialized = true;
+    }
+
+    public static void initializeData() throws ParseException, IOException, InterruptedException {
```

Comment:
Why not just add this init and rollback to those classes that require it (maybe add a BaseAccountTest or ...)? Avoids introducing annotations and using reflection.